extensions/kill-process/package.json

```diff
@@ -76,6 +77,14 @@
       "title": "",
       "type": "checkbox"
     },
+    {
+      "description": "Allow killing multiple processes at once. This keeps the Raycast window open after killing a process",
+      "label": "Allow multiple kills",
+      "name": "allowMultipleKills",
+      "required": false,
+      "title": "Multiple Processes",
```

Comment:
I don't think `title` is needed, for checkboxes it uses the `label`.

Reply:
I thought it would look better with a heading. WIth the title field, it looks like this:
![CleanShot 2023-11-10 at 08 26 26](https://github.com/raycast/extensions/assets/52634785/1fce2092-a6b7-4ecd-8d1a-6d53ce8ec487)

Without it, it would just get smushed into the previous category, Display:
![CleanShot 2023-11-10 at 08 29 15](https://github.com/raycast/extensions/assets/52634785/eae93978-bcb8-4fb5-a407-7d5ee580f196)